doc/getting-started.adoc

```diff
@@ -154,6 +154,41 @@ system configuration. It is recommender to copy and edit the configuration
 files from the `examples` directories if you are basing your configuration off
 of an example.
 
+== Usage with flakes ==
```

Comment:
Minor, but I can't find any confirmation that ending a section title with `==`  is valid.

Anyway, Asciidoctor [does not](https://docs.asciidoctor.org/asciidoc/latest/sections/titles-and-levels/). And other documentation parts do not eithter.